content/waf/install/docker.md

```diff
@@ -424,7 +424,7 @@ Once you have updated your configuration files, you can reload NGINX to apply th
 
 {{< include "waf/install-services-compose.md" >}}
 
-F5 WAF for NGINX should now be operational, and you can move onto [Post-installation checks](#post-installation-checks).
+Review the operational status of the F5 WAF for NGINX following the [Post-installation checks]({{< ref "/waf/install/docker.md#post-installation-checks" >}}).
```

Comment:
```suggestion
You can now review the operational status of F5 WAF for NGINX using the [Post-installation checks]({{< ref "/waf/install/docker.md#post-installation-checks" >}}).
```

The product name was not being used correctly, and the phrasing was unclear, "following the checks" sounds like there's a step after it, instead of directing them towards it as the next sequential step.

If we're changing in-page anchors to be fully inclusive Hugo page references, this will need to be updated in multiple places: this particular piece of text is re-used multiple times on the same page, and there has been no guidance about avoiding inline anchors, so that guidance will also need to be added to process documentation.